docker/nginx.conf

```diff
@@ -12,6 +12,7 @@ http {
     keepalive_timeout  65;
     gzip  on;
     server_tokens off;
+    client_max_body_size 10M;
```

Comment:
I'm thinking if it would make sense to set this to `0`, i.e. disabling the check. What's your thought on this?

Reply:
I'd say it's a matter of opinion more than anything else.

It's easily configurable to suit the needs of someone using this either way. Especially if https://github.com/ninech/netbox-docker/pull/25 is merged.

I think my opinion is that setting it to 10M is a sane default and prevents someone from uploading a huge image by mistake.